components/Card/UserDots.tsx

```diff
@@ -0,0 +1,62 @@
+import { Tooltip } from "@equinor/eds-core-react";
+import React from "react";
+import { UserDot } from "../UserDot";
+import styles from "./UserDots.module.scss";
+
+export function UserDots(props: {
+  users: Array<string>;
+  setVisibleScrim: (any: boolean) => void;
+}): JSX.Element {
+  const { users, setVisibleScrim } = props;
+  const shownUsers = users.length > 3 ? users.slice(0, 3) : users;
+  const numberOtherUsers = users.length > 3 ? users.length - 3 : 0;
+
+  return (
+    <Tooltip title={users.join(", ")}>
+      <button
+        className={styles.buttonUserDots}
+        onClick={(e) => {
+          e.stopPropagation();
+          setVisibleScrim(true);
+        }}
+      >
+        <UserDotsAccordion users={shownUsers} />
+        {!!numberOtherUsers && (
+          <OtherUsersButton numberOtherUsers={numberOtherUsers} />
+        )}
+      </button>
+    </Tooltip>
+  );
+}
+
+export function UserDotsAccordion(props: { users: string[] }): JSX.Element {
+  const users = props.users.reverse();
+
+  const numberOfUserDots =
```

Comment:
Better to name this based on what it sets. 
I guess it is setting the button-width?

Adding css classNames like this is complicated. Inline JSX styling is allowed if it makes it easier.

Consider changing to:
```tsx
const width = users.length <= 3 ? 24 * users.length : 24;

  return (
    <div style={{ width }} className={`${styles.containerUserDots}`}>
```